recipes/discovar/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -x -e
+
+export CC=${PREFIX}/bin/gcc
```

Comment:
I'll commit a build without it. I suppose you're right. Is ${PREFIX}/bin already the first entry in PATH?